Formula/stress-ng.rb

```diff
@@ -0,0 +1,18 @@
+class StressNg < Formula
+  desc "Stress test a computer system in various selectable ways"
+  homepage "http://kernel.ubuntu.com/~cking/stress-ng/"
+  url "http://kernel.ubuntu.com/~cking/tarballs/stress-ng/stress-ng-0.09.06.tar.xz"
+  sha256 "59c09c7cccb34222dc124ce8622a0ac73fe9391ba16a4a561267ca55c02f2520"
+
+  depends_on :macos => :sierra
+
+  def install
+    inreplace "Makefile", "/usr", prefix
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/stress-ng", "-c 1", "-t 1"
```

Comment:
Can we have this verify something beyond just a zero exit code?

Reply:
We could look to assert the output matches. Would that be sufficient for what you're after here?

Reply:
Addressed in 8eae3e3